dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM node:8-alpine AS build
+
+# Create the app directory
+WORKDIR /app
+
+# Copy package json
+COPY package*.json ./
+
+# COPY yarn.lock
```

Comment:
Remove Yarn, because you don't have any tests!

Reply:
no because it needed yarn file for the depencies

Reply:
Why is your yarn.lock file empty then??

That means you aren't using yarn

Reply:
removed